### ISSUE TITLE
False negative in Layout/SpaceBeforeFirstArg

### ISSUE THREAD

**Issue by tatsuyafw** [rubocop-hq/rubocop#6615]
## Expected behavior

The following code registers an offense about `Layout/SpaceBeforeFirstArg`.

```ruby
foobar :foo
foobarbaz :foo
foo    :foo # here

foobar :foo
foobarbaz :foo
foo :foo
```

## Actual behavior

`Layout/SpaceBeforeFirstArg` cop does not register an offense.

## Steps to reproduce the problem

```sh
$ cat foo.rb
foobar :foo
foobarbaz :foo
foo    :foo

foobar :foo
foobarbaz :foo
foo :foo
$
```

```
$ rubocop --only "Layout/SpaceBeforeFirstArg" foo.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
$
```

## RuboCop version

```
$ rubocop -V
0.61.1 (using Parser 2.5.3.0, running on ruby 2.6.0 x86_64-darwin18)
```

**Comment by tatsuyafw** [comment]
@Drenmi 

Hi.
After investigating the cause, I've thought this issue seems to be not a bug.

The comments for AllowForAlignment config of `Layout/SpaceBeforeFirstArg` says:

https://github.com/rubocop-hq/rubocop/blob/92605c92018c14933042d53790f894323384f124/config/default.yml#L983-L986

Therefore, rubocop ignores empty lines and regards the line 3 and line 5 of the following code as aligned.

```ruby
1  foobar :foo
2  foobarbaz :foo
3  foo    :foo
4
5  foobar :foo
6  foobarbaz :foo
7  foo :foo
```

Is it better to close this issue?
